cmd/kube-proxy/app/server_others.go

```diff
@@ -193,6 +195,14 @@ func newProxyServer(
 		if err != nil {
 			return nil, fmt.Errorf("unable to create proxier: %v", err)
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) {
```

Comment:
Shouldn't this if statement surround line#75 based on the # of ciders provided?

Reply:
No, line#175 is original unchanged code, it creates a proxyier with the ClusterFamily, ipv4 or ipv6.

In line#198, if the feature-gate is set, a proxier of the "other" family is created and also the "meta-proxier" that dispatches between the 2 proxier instances.

Reply:
In the current code that is. After reading https://github.com/kubernetes/kubernetes/pull/80627#pullrequestreview-277510712 I see your meaning. I will refactor this.